run/cloudrun-pubsub/Dockerfile

```diff
@@ -21,3 +27,5 @@ COPY --from=builder /app/target/cloudrun-pubsub-*.jar /cloudrun-pubsub.jar
 
 # Run the web service on container startup.
 CMD ["java","-Djava.security.egd=file:/dev/./urandom","-Dserver.port=${PORT}","-jar","/cloudrun-pubsub.jar"]
```

Comment:
Why are we doing this?  The default is a bit more secure.

Reply:
It needs to be able to run based on the PORT env var. Also the security flag is for faster loading of the springboot app. Do you suggest something else?

Reply:
This is being copied around in all knative and Cloud Run Dockerfiles. It seems as though while less secure, it is regarded as a recommended performance optimization in containers.

* https://docs.oracle.com/cd/E13209_01/wlcp/wlss30/configwlss/jvmrand.html
* https://spring.io/guides/gs/spring-boot-docker/

I am happy to defer to Java experts on changing this here and the knative [hellworld-java-spring sample](https://github.com/knative/docs/blob/master/docs/serving/samples/hello-world/helloworld-java-spring/Dockerfile).

Reply:
You might wish to comment in the file why this is being done (performance) and that if security is absolutely required (ie, you make a SSL request), you might wish to skip it.

Reply:
That's a good idea. Averi do you want to wordsmith something for this and the knative helloworld-java-spring sample?

Reply:
`-Djava.security.egd=file:/dev/./urandom` is being done to improve [startup](https://spring.io/guides/gs/spring-boot-docker/) [performance](https://stackoverflow.com/questions/137212/how-to-solve-slow-java-securerandom). This is acceptable for many applications. (especially when paired with Cloud KMS).  If your application does it's own key generation, such as for SSL, and you require greater security, you may prefer to not set `java.security.egd`.

feel free to suggest alternatives.  (Note - I didn't include the first link as it really didn't mention java.security.egd) @grayside

Reply:
Doing more research, https://spring.io/guides/gs/spring-boot-docker/ says that it's not necessary anymore for Tomcat 7+. I'm going to remove.

Reply:
Averi did some benchmarks, and found it doesn't help startup of spring boot apps anymore.  So, we might mention it in the docs under Tips for improving application startup time -- Java, but let's pull it from samples.

Reply:
done.